README.md

```diff
@@ -74,6 +74,8 @@ manually if your distribution does not provide a package.
 
 First, make sure you have the required __dependencies__:
 - build:
+	- autopoint (for bootstrapping)
```

Comment:
Yep maybe we should remain simple. Developers can handle these build dependencies by themselves, these are just details.